ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -706,8 +706,19 @@ final class Metrics(val registry: MetricRegistry) {
     object HttpJsonApi {
       private val Prefix: MetricName = daml.Prefix :+ "http_json_api"
 
-      val httpRequest: Timer = registry.timer(Prefix :+ "http_request")
-      val websocketRequest: Counter = registry.counter(Prefix :+ "websocket_request")
+      // Meters how long processing of a request takes
+      val httpRequestTimer: Timer = registry.timer(Prefix :+ "http_request_timing")
+      // Meters http requests throughput
```

Comment:
totally fine to do this in a separate PR but I’d probably just replace this by a meter for queries. I don’t think seeing requests of different types grouped together is very useful.

Reply:
I probably adjust this in the next PR where I want to add the timers for the queries.